packages/monosize/src/commands/measure.mts

```diff
@@ -45,20 +46,17 @@ async function measure(options: MeasureOptions) {
   const measurements: BuildResult[] = [];
 
   for (const preparedFixture of preparedFixtures) {
-    measurements.push(
-      await buildFixture({
```

Comment:
`buildFixture` also contains`outputPath` in return, we don't want to have it in JSON file.